src/style.css

```diff
@@ -1,3 +1,3 @@
 .completed {
-    text-decoration: line-through;
-}
\ No newline at end of file
+  text-decoration: line-through;
+}
```

Comment:
- [ ] it would be better to style your to-do list as in the given [video ](https://www.youtube.com/watch?v=AcUd-_Yjjqg) 
- Yours 
<img width="213" alt="Screenshot 2023-02-23 at 14 45 02" src="https://user-images.githubusercontent.com/61790587/220925033-07324fb4-f52d-4d4e-83ac-68675fbe9f20.png">
- Design 
<img width="812" alt="Screenshot 2023-02-23 at 14 44 23" src="https://user-images.githubusercontent.com/61790587/220925065-1b2ea18c-0fe1-46bf-8fe0-a86ae8e20dfe.png">